payroll/payment_gateway/payment_gateway_config.py

```diff
@@ -0,0 +1,46 @@
+import importlib
+
+from payroll.apps import PayrollConfig
+
+
+class PaymentGatewayConfig:
+    def __init__(self):
+        self.gateway_base_url = PayrollConfig.gateway_base_url
+        self.endpoint_payment = PayrollConfig.endpoint_payment
+        self.endpoint_reconciliation = PayrollConfig.endpoint_reconciliation
+        self.api_key = PayrollConfig.payment_gateway_api_key
+        self.basic_auth_username = PayrollConfig.payment_gateway_basic_auth_username
+        self.basic_auth_password = PayrollConfig.payment_gateway_basic_auth_password
+        self.timeout = PayrollConfig.payment_gateway_timeout
+        self.auth_type = PayrollConfig.payment_gateway_auth_type
```

Comment:
Could be good to add a comment or notify somewhere that in order to make it work Payroll module needs to go before this module in openimis.json